updateagent/agent.py

```diff
@@ -152,7 +152,7 @@ def __repr__(self):
         return "LooseVersion ('%s')" % str(self)
 
     def __cmp__(self, other):
-        if isinstance(other, StringType):
```

Comment:
Would we need to supply other operators since `cmp` is to be treated as if it's not there in Python3?